src/entry/data-provider/index.ts

```diff
@@ -31,6 +35,32 @@ export const dataProvider = async (request: DataProviderPayload): Promise<DataPr
 
   const trackingBranch = await getTrackingBranchName(groupToken, projectId, defaultBranch);
 
+  const backfillData: BackfillData = {
+    builds: [],
+    deployments: [],
+    metrics: {
+      mrCycleTime: 0,
+      openMergeRequestsCount: 0,
+    },
+  };
+
+  try {
+    const {
+      builds,
+      deployments,
+      metrics: { mrCycleTime, openMergeRequestsCount },
+    } = await getBackfillData('groupToken', projectId, projectName, trackingBranch);
+
+    backfillData.builds = builds;
+    backfillData.deployments = deployments;
+    backfillData.metrics.mrCycleTime = mrCycleTime;
+    backfillData.metrics.openMergeRequestsCount = openMergeRequestsCount;
+  } catch (err) {
+    const invocationErrorOptions = { backoffTimeInSeconds: 0 };
```

Comment:
@kbielaski What is the standard number for backoff time here?

Reply:
I believe the default value for a user's gitlab account is 3600. We should be good to go with that number.